sdk/deepbook-v3/src/client.ts

```diff
@@ -176,23 +184,22 @@ export class DeepBookClient {
 
 		const pool = this.#config.getPool(poolKey);
 
-		return this.#signAndExecuteCommand(
-			this.#deepBook.placeLimitOrder(
-				pool,
-				balanceManager,
-				clientOrderId,
-				price,
-				quantity,
-				isBid,
-				expiration,
-				orderType,
-				selfMatchingOption,
-				payWithDeep,
-			),
+		return this.#deepBook.placeLimitOrder(
+			pool,
+			balanceManager,
+			clientOrderId,
+			price,
+			quantity,
+			isBid,
+			expiration,
+			orderType,
+			selfMatchingOption,
+			payWithDeep,
+			tx,
 		);
 	}
 
-	placeMarketOrder(params: PlaceMarketOrderParams) {
+	placeMarketOrder(params: PlaceMarketOrderParams, tx: Transaction = new Transaction()) {
```

Comment:
I like that you have combined params into input object, but we should be consistent about doing this across all the methods of the sdk